webapp/channels/src/components/admin_console/team_channel_settings/group/group_list.tsx

```diff
@@ -66,9 +67,21 @@ const GroupList = ({
         <AbstractList
             header={<Header/>}
             renderRow={renderRow}
+            emptyListText={isModeSync ? messages.emptyListModeSync : messages.emptyList}
             {...restProps}
         />
     );
 };
 
+const messages = defineMessages({
```

Comment:
I like this better before the component (everything is defined before used) but that is just about taste.

Reply:
Leave this as-is